test/helpers/sharedBootstrap.js

```diff
@@ -29,6 +48,17 @@ global.request_auth = app =>
     .set('x-forwarded-proto$', 'https')
     .set('x-real-ip', '127.0.0.1');
 
+global.testConfig = {
+  initializeBarrelsFixtures: true
+};
+
+global.createRequest = function (payload, path, res) {
+  return request_auth(sails.hooks.http.app)
+    .post(path)
+    .send(payload)
+    .expect(res || 201);
```

Comment:
not sure you'd necessarily want this logic of expecting the response to be what you want it to be `or` a status code?